Presentation.Web/Tests/it-system/Catalog/Migration.System.e2e.spec.ts

```diff
@@ -0,0 +1,106 @@
+            .then(() => cataHelper.createCatalog(systemNameFrom))
+            .then(() => cataHelper.createCatalog(systemNameTo))
+            .then(() => toggleSystemActivation(systemNameFrom))
+            .then(() => openMigrationOnSpecificSystem(systemNameFrom))
+            .then(() => waitForModalWindowAnimation(constants.moveSystemButton))
+            .then(() => expect(element(cssHelper.byDataElementType(constants.moveSystemButton)).isPresent()).toBe(true))
+            .then(() => element(cssHelper.byDataElementType(constants.moveSystemButton)).click())
+            .then(() => waitForModalWindowAnimation(constants.consequenceButton))
+            .then(() => expect(element(cssHelper.byDataElementType(constants.consequenceButton)).isPresent()).toBe(true))
+            .then(() => select2SearchForSystem(systemNameTo))
+            .then(() => waitForSelect2DataAndSelect())
+            .then(() => element(cssHelper.byDataElementType(constants.consequenceButton)).click())
+            .then(() => waitForModalWindowAnimation(constants.startMigrationButton))
+            .then(() => expect(element(cssHelper.byDataElementType(constants.startMigrationButton)).isDisplayed()).toBe(true))
+            .then(() => element(cssHelper.byDataElementType(constants.startMigrationButton)).click())
+            .then(() => expect(element(cssHelper.byDataElementType(constants.startMigrationButton)).isDisplayed()).toBe(false));
+    });
+
+    function createItSystemName() {
+        return "It-System" + new Date().getTime();
+    }
+
+    function toggleSystemActivation(name: string) {
+        return pageObject.getPage()
+            .then(() => element(by.xpath('//*/tbody/*/td/a[text()="' + name + '"]/parent::*/parent::*//*/button')).click());
+    }
+
+    function openMigrationOnSpecificSystem(name: string) {
+        return pageObject.getPage()
+            .then(() => element(by.xpath('//*/tbody/*/td/a[text()="' + name + '"]/parent::*/parent::*//*/a[@data-element-type="usagesLinkText"]')).click());
+    }
+
+    function waitForModalWindowAnimation(name: string) {
+        var EC = protractor.ExpectedConditions;
+        return browser.wait(EC.visibilityOf(element(cssHelper.byDataElementType(name))),
+            20000);
+    }
+
+    function waitForSelect2DataAndSelect() {
+        var EC = protractor.ExpectedConditions;
+        return browser.wait(EC.visibilityOf(element(by.className("select2-result-label"))), 20000)
+            .then(() => element(by.className("select2-input")).sendKeys(protractor.Key.ENTER));
+    }
+
+    function select2SearchForSystem(name: string) {
+        return element(by.className("select2-choice")).click()
+            .then(() => element(by.className("select2-input")).click())
+            .then(() => element(by.className("select2-input")).sendKeys(name));
+    }
+
+    function checkMigrationButtonVisibility(isVisible: boolean) {
+        return pageObject.getPage()
```

Comment:
Måske getPage i virkeligheden skulle være en del af BeforeEach, for det er jo det første den skal gøre